### PR TITLE
Viite 2470 node tool performance

### PR DESCRIPTION
**Pull request by oskarmalmstrom**

### CODE REVIEW

File: viite-UI/src/view/NodeLayer.js [hunk 1]
```diff
@@ -569,7 +569,6 @@
       } else if (layer === 'node') {
         setGeneralOpacity(1);
         addInteractions();
-        showLayer();
```
Comment (oskarmalmstrom) [inline viite-UI/src/view/NodeLayer.js:572]: showLayer is already called in MapView.js line 107

File: viite-UI/src/view/NodeForm.js [hunk 2]
```diff
@@ -409,8 +409,8 @@
     };
 
     var closeNode = function (cancel) {
+      eventbus.off('change:nodeName change:nodeTypeDropdown change:nodeStartDate junction:validate junction:setCustomValidity');
```
Comment (peltsi212) [inline viite-UI/src/view/NodeForm.js:412]: This list contains now more events than before. I think eventbus.off should be called for all places where there is eventbus.on for example node:selected is still missing

Comment (oskarmalmstrom) [reply viite-UI/src/view/NodeForm.js:412]: The listener for node:selected is only created when `bindEvents()` is called on initialization so there is no need to remove the listener. But for the listeners created inside node:selected it's true, I will add them.

Comment (oskarmalmstrom) [reply viite-UI/src/view/NodeForm.js]: ...but you are right, the node:selected listener still needs to be removed. I will correct that.

**Pull request merged**